examples/project-exporters/vue-basic/index.ts

```diff
@@ -34,7 +35,7 @@ const writeToDisk = async (
 //   console.log(JSON.stringify(result, null, 2))
 // }
 
-console.log(UIDLValidators.validateProject(projectJson))
```

Comment:
this is just for testing purposes so it can remain as a console.log to see the results of the validation when we run the generators (we can add a ignore comment)